guide/getting-started/hardware.md

```diff
@@ -71,14 +76,13 @@ You may already be familiar with key fobs from your bank or work. They are small
    height = 400
 %}
 
-Bitcoin hardware wallets play a similar role but with several noteworthy differences. Like key fobs, they securely store an authentication method and isolate it from the Internet, networks, and other devices. In bitcoin, these are the seed, associated keys, and functions that require interaction with them.
+Bitcoin hardware wallets, also known as external signers, act like Bitcoin centric security keys. They isolate the recovery phrase, private keys, and other sensitive data from the internet and other devices.
```

Comment:
This isn't entirely correct. Hardware wallets do not isolate sensitive data from the `internet`. As the matter a fact, majoirty of hardware wallets will force users to use third-party web-wallets that rely on their servers in order to send a transactions, leaking all sensitive information to the manufacturer. 

What most hardware wallets isolate is private key, and they just spit out the signature, which means they sign a transaction on a secure chip element, but any other sensitive data may be leaked.

For example xpub is in most cases leaked, and that's indeed a sensitive data.

And reading the paragraph below it, we may not even need this paragraph.

```suggestion
Bitcoin hardware wallets, also known as external signers, act like Bitcoin centric security keys. They isolate the recovery phrase and private keys from the internet and other devices.
```